packages/react/CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.11.3](https://github.com/digdir/designsystem/compare/@digdir/design-system-react@0.11.1...@digdir/design-system-react@0.11.3) (2023-05-04)
+
+### Bug Fixes
+
+- **InputWrapper:** Make input borders clickable ([#282](https://github.com/digdir/designsystem/issues/282)) ([d35df3f](https://github.com/digdir/designsystem/commit/d35df3f852359a0e21e9d05f8259158036db1206))
+- **Select:** Collapse when arrow button is clicked ([#336](https://github.com/digdir/designsystem/issues/336)) ([254628a](https://github.com/digdir/designsystem/commit/254628a7918dc037643b7157b40ad3e7031daeae))
+- **Select:** fix aria-activedecendant attribute ([#291](https://github.com/digdir/designsystem/issues/291)) ([0ed8958](https://github.com/digdir/designsystem/commit/0ed89584083a66d20d4f37570825f023fc40c24a))
+- **Select:** Make event listeners local ([#299](https://github.com/digdir/designsystem/issues/299)) ([1439820](https://github.com/digdir/designsystem/commit/14398207f95816f13ff9133913c0ee40080c25f9))
```

Comment:
What does this fix?

Reply:
I'll just remove it because the bug it fixes was introduced after the last release.